code/mob/living/critter/maneater.dm

```diff
@@ -27,111 +27,303 @@
+	..()
+
+
+
+/mob/living/critter/plant/maneater/seek_food_target(var/range = 7)
+	. = list()
+	for (var/obj/item/reagent_containers/food/snacks/ingredient/meat/potential_meat in view(range, get_turf(src)))
+		//no fish meat, no synthmeat. chickens nuggets are ok, though.
+		if (istype(potential_meat, /obj/item/reagent_containers/food/snacks/ingredient/meat/fish)) continue
+		if (istype(potential_meat, /obj/item/reagent_containers/food/snacks/ingredient/meat/synthmeat)) continue
+		. += potential_meat
+
+
+/mob/living/critter/plant/maneater/seek_target(var/range = 9)
+	. = ..()
+	if ((length(.) > 0) && prob(10))
+		if (!ON_COOLDOWN(src, "maneater_snarling", 15 SECONDS))
+			playsound(src.loc, 'sound/voice/maneatersnarl.ogg', 50, 1, channel=VOLUME_CHANNEL_EMOTE)
+			src.visible_message("<span class='alert'><B>[src]</B> snarls!</span>")
+
+/mob/living/critter/plant/maneater/seek_scavenge_target(var/range = 5)
+	. = list()
+	var/target_being_devoured = FALSE
+	for (var/mob/living/carbon/human/checked_human in view(range, get_turf(src)))
+		target_being_devoured = FALSE
+		for(var/mob/living/critter/plant/maneater/checked_maneater in get_turf(checked_human))
+			if (checked_maneater != src)
+				target_being_devoured = TRUE
+		if (!(target_being_devoured) && isdead(checked_human) && checked_human.decomp_stage <= 3 && !checked_human.bioHolder?.HasEffect("husk"))
+			//is dead, isn't a skeleton, isn't a grody husk, isn't occupied by another maneater
+			. += checked_human
+
+/mob/living/critter/plant/maneater/can_critter_scavenge()
+	var/datum/targetable/critter/checked_ability = src.abilityHolder.getAbility(/datum/targetable/critter/maneater_devour)
+	if (checked_ability.disabled || !checked_ability.cooldowncheck()) return FALSE
+	return can_act(src,TRUE)
+
+
+
+/mob/living/critter/plant/maneater/critter_attack(mob/target)
+	// first we check if our maneater is munching on something
+	var/datum/targetable/critter/devour_ability = src.abilityHolder.getAbility(/datum/targetable/critter/maneater_devour)
+	if (!(src in actions.running))
+		//first, we check if another maneater is on that persons tile. This way, we don't have food fights between maneaters
+		var/target_being_devoured = FALSE
+		for(var/mob/living/critter/plant/maneater/checked_maneater in get_turf(target))
+			if (checked_maneater != src)
+				target_being_devoured = TRUE
+		//if the target is unconscious, being eaten by another maneater and we are unable to eat them, we gotta wack them a bit
+		if(!target_being_devoured && (isunconscious(target) || isdead(target)) && ishuman(target) && !devour_ability.disabled && devour_ability.cooldowncheck())
+			//we we grab and devour our target :)
+			return src.grab_and_devour(target, devour_ability)
+
+		else
+			//we want to nibble on them with out right hand
+			src.set_a_intent(INTENT_HARM)
+			src.set_dir(get_dir(src, target))
+			src.active_hand = 2
+
+			var/list/params = list()
+			params["right"] = TRUE
```

Comment:
?

Reply:
That was copied out of ``/mob/living/critter/small_animal/pikaia``. But looking from the parent of ``critter_attack`` this is not needed at all. Gotta throw it out then.